modules/sdk-coin-bsc/src/lib/resources.ts

```diff
@@ -0,0 +1,22 @@
+import EthereumCommon from '@ethereumjs/common';
+import { coins, EthereumNetwork } from '@bitgo/statics';
+
+export const testnetCommon = EthereumCommon.forCustomChain(
+  'mainnet',
+  {
+    name: 'testnet',
+    networkId: 1,
```

Comment:
this wont work for BSC -- networkId; 1 is for ETH, bsc will be 56 as per https://chainlist.org/chain/56 for mainnet, and 97 for testnet https://chainlist.org/chain/97

Reply:
ah crap maybe networkId is fine and these should just be for chainid -- i'd just make sure we add bsc and tbsc to bitgojs/statics properly

Reply:
Changed so that networkId and chainId are the same